src/main/java/net/atos/zac/documentcreatie/converter/DocumentCreatieDataConverter.java

```diff
@@ -144,14 +144,14 @@ private ZaakData createZaakData(final Zaak zaak) {
                 .map(RolOrganisatorischeEenheid::getNaam)
                 .ifPresent(groep -> zaakData.groep = groep);
 
-        zgwApiService.findBehandelaarForZaak(zaak)
+        zgwApiService.findBehandelaarMedewerkerRoleForZaak(zaak)
                 .map(RolMedewerker::getNaam)
                 .ifPresent(behandelaar -> zaakData.behandelaar = behandelaar);
```

Comment:
This is a side-effect pattern and should be avoided. Preferably to turn this around into something like:
```
zaakData.behandelaar =  zgwApiService.findBehandelaarMedewerkerRoleForZaak(zaak)
                .map(RolMedewerker::getNaam)
                .orElse(null);
```
(not tested, but there should be a method like that)

Reply:
Definitely this Java code could be improved but I don't want to do that in this PR. I would much rather improve this code when we actually touch it (and not just renaming of a method like here) and preferably first convert it to Kotlin. :-)